middleware/hbs-helpers.js

```diff
@@ -0,0 +1,38 @@
+const hbs = require('express-hbs')
+const getOcticons = require('./register-octicons')
+const lobars = require('lobars')
+
+class HandlebarsHelpers {
```

Comment:
Why does this need to be a class rather than an object?

Reply:
Just convenience during scaling. It could be changed to just to functions.